packages/ui/SurveysList/components/SurveyFilters.tsx

```diff
@@ -1,75 +1,59 @@
 import { ChevronDownIcon, Equal, Grid2X2, Search, X } from "lucide-react";
-import { useEffect, useState } from "react";
+import { useState } from "react";
+import { useDebounce } from "react-use";
 
-import { TSurvey } from "@formbricks/types/surveys";
+import { TSurveyFilters } from "@formbricks/types/surveys";
 
+import { initialFilters } from "..";
 import { Button } from "../../Button";
-import { Checkbox } from "../../Checkbox";
-import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from "../../DropdownMenu";
+import { DropdownMenu, DropdownMenuContent, DropdownMenuTrigger } from "../../DropdownMenu";
 import { TooltipRenderer } from "../../Tooltip";
+import { SortOption } from "./SortOption";
+import { SurveyFilterDropdown } from "./SurveyFilterDropdown";
 
 interface SurveyFilterProps {
-  surveys: TSurvey[];
-  setFilteredSurveys: (surveys: TSurvey[]) => void;
   orientation: string;
   setOrientation: (orientation: string) => void;
-  userId: string;
+  surveyFilters: TSurveyFilters;
+  setSurveyFilters: React.Dispatch<React.SetStateAction<TSurveyFilters>>;
 }
-interface TFilterOption {
+export interface TFilterOption {
```

Comment:
As this is been used more than once, can we move this to `types` package and use zod as well ? (similar for `TSortOption`) 😊